.github/workflows/ci-cd.yml

```diff
@@ -22,28 +22,41 @@ jobs:
       - run: yarn -s test:unit
 
   publish:
-    needs: unit-tests
+    # needs: unit-tests
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0
       - uses: actions/setup-node@v1
-      - run: yarn --frozen-lockfile
-      - run: yarn -s dripip preview-or-pr
+      - name: Install deps
+        run: yarn --frozen-lockfile
+      - name: Make release
+        run: |
+          yarn -s dripip preview-or-pr --json > result.json
+          cat result.json | jq '.'
         env:
           NPM_TOKEN: ${{secrets.NPM_TOKEN}}
+      - name: Save publish result
```

Comment:
Need this approach b/c the git commit in a workflow is not visible to later jobs.